components/low_power_mode/src/low_power_mode.h

```diff
@@ -0,0 +1,8 @@
+#ifndef LOW_POWER_MODE_H
+#define LOW_POWER_MODE_H
+
+void low_power_mode_enter_deep_sleep(void);
```

Comment:
If It's a public function - API - then it's a good practice to add some comments what this function does. Please consider usage of Doxygen format.